app/styles/module/_qualaroo.scss

```diff
@@ -0,0 +1,123 @@
+$qualaroo-height: 280px;
+$qualaroo-width-small: 310px;
+$qualaroo-width-large: 350px;
+
+.mobile-wiki {
+	div#qual_ol {
```

Comment:
a note about that these are coming from quoalaroo might make it more obvious, what that actually is